README.md

```diff
@@ -137,9 +138,9 @@ React Native showcase.
 
 ### Make the change
 
-1. Follow the "Running locally" instructions
-1. Save the files and check in the browser. Some changes may require a server
-   restart.
+1. Follow the "Running locally" instructions 
+1. `open http://localhost:3000/react-native/versions.html` to see other versions.
```

Comment:
I'd say something like:

>1. Follow the "Running locally" instructions 
>1. Save the files and check in the browser. Some changes may require a server restart.
>1. Changes to `/docs` will only be visible in the latest version of the documentation (master). 
>`open http://localhost:3000/react-native/versions.html` to see other versions.

The files should be saved before checking versions and this add clarification on why we should see other versions.

What do you think?

Reply:
Yup makes sense. I added an update with your suggestion.